apps/frontend/src/components/global/groups/Users.vue

```diff
@@ -123,8 +131,11 @@ export default class Users extends Vue {
 
     ServerModule.allUsers.forEach((user) => {
       if(!currentUserIds.includes(user.id)) {
+        const text = user.title !== null
+          ? `${user.firstName || ''} ${user.lastName || ''} (${user.email}, ${user.title})`
+          : `${user.firstName || ''} ${user.lastName || ''} (${user.email})`
```

Comment:
Instead of building the string twice here, can you do:

```javascript
let text = `${user.firstName || ''} ${user.lastName || ''} (${user.email}, ${user.title})`
if(user.title !== null) {
	text += `, ${user.title})`;
}
```